.github/workflows/library-nginx1.15.yaml

```diff
@@ -47,6 +47,34 @@ jobs:
         arch: ${{ matrix.arch }}
         push: false
         output: oci://index.unikraft.io/unikraft.org/nginx:1.15
+        after: |
+          # GitHub Actions can only executed with QEMU (via emulation mode)
+          if [[ ${{ matrix.plat }} != "qemu" ]]; then
+            exit 0;
+          fi
+
+          export KRAFTKIT_LOG_LEVEL=debug
+          export KRAFTKIT_RUNTIME_DIR=/github/workspace/.kraftkit
+
+          set -xe;
+
+          kraft run \
+            --detach \
+            --disable-acceleration \
+            --plat ${{ matrix.plat }} \
+            --arch ${{ matrix.arch }} \
+            --name nginx-${GITHUB_RUN_ID} \
+            --port 8080:80 \
+            index.unikraft.io/unikraft.org/nginx:1.15
```

Comment:
This should be rethought as the action now supports running things

Still, we can merge this if ok